build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+  id 'java'
+  id 'eclipse' // optional (to generate Eclipse project files)
+  id 'idea' // optional (to generate IntelliJ IDEA project files)
+}
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+  testCompile('org.junit.jupiter:junit-jupiter-api:5.3.1')
```

Comment:
I think we should check that version 5.3.1 is the latest released version.

Reply:
[being another person] Yeah, actually, this is the latest version. So, all good.